src/Chapter2.hs

```diff
@@ -740,8 +752,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: (Eq a, Num a) => a -> [[a]] -> [[a]]
```

Comment:
The `Num` constraint is redundant here. Note, that `elem` needs only `Eq` for this function to work. We don't require people to write very polymorphic functions here, so by saying _list of lists of numbers_, we mean that a type signature like `Int -> [[Int]] -> [[Int]]` will be enough for this task. Nevertheless, nice work on specifying a polymorphic type signature :+1: 
```suggestion
contains :: (Eq a) => a -> [[a]] -> [[a]]
```